src/containers/ListingPage/ListingPage.shared.js

```diff
@@ -138,21 +142,37 @@ export const handleContactUser = parameters => () => {
  * @param {Object} parameters all the info needed to create inquiry.
  */
 export const handleSubmitInquiry = parameters => values => {
-  const { history, params, getListing, onSendInquiry, routes, setInquiryModalOpen } = parameters;
-  const listingId = new UUID(params.id);
-  const listing = getListing(listingId);
-  const { message } = values;
+  const {
+    history,
+    params,
+    getListing,
+    onSendInquiry,
+    routes,
+    setInquiryModalOpen,
+    currentUser,
+  } = parameters;
```

Comment:
Check my other comments. My suggestion is that you don't make changes to this function.

Reply:
Updated and moved the check to the handleContactUser function!